Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1186,67 +1185,64 @@ class BrowserViewController: UIViewController {
         return tabManager.selectedTab?.webView?.becomeFirstResponder() ?? false
     }
 
-    func reloadTab(){
-        if(homePanelController == nil){
+    func reloadTab() {
+        if(homePanelController == nil) {
             tabManager.selectedTab?.reload()
         }
     }
 
-    func goBack(){
-        if(tabManager.selectedTab?.canGoBack == true && homePanelController == nil){
+    func goBack() {
+        if(tabManager.selectedTab?.canGoBack == true && homePanelController == nil) {
             tabManager.selectedTab?.goBack()
         }
     }
-    func goForward(){
-        if(tabManager.selectedTab?.canGoForward == true && homePanelController == nil){
+    func goForward() {
+        if(tabManager.selectedTab?.canGoForward == true && homePanelController == nil) {
             tabManager.selectedTab?.goForward()
         }
     }
 
-    func findOnPage(){
-        if(homePanelController == nil){
+    func findOnPage() {
+        if(homePanelController == nil) {
             tab( (tabManager.selectedTab)!, didSelectFindInPageForSelection: "")
         }
     }
 
-    func selectLocationBar(){
+    func selectLocationBar() {
         scrollController.showToolbars(animated: true)
         urlBar.tabLocationViewDidTapLocation(urlBar.locationView)
     }
 
-    func newTab(){
+    func newTab() {
         openBlankNewTabAndFocus(isPrivate: false)
     }
-    func newPrivateTab(){
+    func newPrivateTab() {
         openBlankNewTabAndFocus(isPrivate: true)
     }
 
-    func closeTab(){
-        if(tabManager.tabs.count > 1){
-            tabManager.removeTab(tabManager.selectedTab!);
-        }
-        else{
+    func closeTab() {
+        if(tabManager.tabs.count > 1) {
```

Comment:
I'm surprised that it doesn't throw a linter error for the lack of whitespace between the if and (. Probably worth just cleaning up the brackets for the methods below too.